lib/charms/mongodb/v1/mongodb_backups.py

```diff
@@ -733,3 +754,20 @@ def process_pbm_error(self, pbm_status: Optional[_StrOrBytes]) -> str:
         elif "status code: 301" in error_message:
             message = "s3 configurations are incompatible."
         return message
+
+    def _needs_provided_remap_arguments(self, backup_id: str) -> bool:
+        """Returns true if remap arguments are needed to perform a restore command."""
+        pbm_status = self.charm.run_pbm_command(["status", "--out=json"])
+        pbm_status = json.loads(pbm_status)
+        backups = pbm_status["backups"]["snapshot"] or []
```

Comment:
is it always guaranteed that `pbm_status` will contains these keys and that `pbm_status["backups"]` will contain "snapshot" key or it is possible that this will trigger KeyError exception?

Reply:
At the point in which this is executed it should always have both. But I made Mehdi's suggestion for "snapshot"